File: src/frontend/src/types/comment.ts
export interface CommentBo {
    uid: string;
    timestamp: string;
    creatorUid: string;
    postUid: string;
    content: string;
    profileIconColor: string;
    profileIcon: string;
    displayName: string;
    inEdit: boolean;
}

export interface CreateCommentDto {
    postUid: string;
    creatorUid: string;
    content: string;
}

export interface UpdateCommentDto {
    content: string;
}

File: src/backend/Hermes.SituationRoom.Data/Entities/Comment.cs
using System;
using System.Collections.Generic;

namespace Hermes.SituationRoom.Data.Entities;

public partial class Comment
{
    public Guid Uid { get; set; }

    public DateTime Timestamp { get; set; }

    public string Content { get; set; }

    public Guid CreatorUid { get; set; }

    public Guid PostUid { get; set; }

    public virtual User CreatorU { get; set; }

    public virtual Post PostU { get; set; }
}
